nixos/tests/paperless.nix

```diff
@@ -4,6 +4,7 @@ import ./make-test-python.nix ({ lib, ... }: {
 
   nodes.machine = { pkgs, ... }: {
     environment.systemPackages = with pkgs; [ imagemagick jq ];
+    time.timeZone = "America/Los_Angeles";
```

Comment:
This has no effect on the test outcome.
Either check that the timezone is picked up by paperless or remove this line (I'd prefer just removing it).